packages/modal-content/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as ModalContentState } from './ModalContentState'
```

Comment:
Maybe, better ways is move ModalContentState and ModalButton to new module called @lidofinance/modal-additionals (helpers).

When i import @lidofinance/modal i am waiting to import  only modals without additionals (helpers) components.

Reply:
It's also a good solution, I like it. But the current package already exports [extra components](https://github.com/lidofinance/ui/blob/main/packages/modal/src/index.ts). It seems it would be great to have one common approach for such cases. If we decide to go this way, it's worth refactoring the current `Modal` package. And check other packages that also export more than one component.